src/app/components/drawer/Projects/index.js

```diff
@@ -3,28 +3,35 @@ import React from 'react';
 import { QueryRenderer, graphql } from 'react-relay/compat';
 import Relay from 'react-relay/classic';
 import ProjectsComponent from './ProjectsComponent';
+import FeedsComponent from './FeedsComponent';
 
-const renderQuery = ({ error, props }) => {
+const renderQuery = ({ error, props, drawerType }) => {
   if (!error && props) {
     const feedsCreated = props.team.feeds.edges.map(f => f.node).filter(f => f.team_id === props.team.dbid);
     const feedsJoined = props.team.feed_teams.edges.map(ft => ft.node).filter(ft => !feedsCreated.find(f => f.dbid === ft.feed_id));
     const feedsInvited = props.me.feed_invitations.edges.map(f => f.node).filter(fi => fi.state === 'invited');
     const feeds = [].concat(feedsCreated, feedsJoined, feedsInvited);
     return (
-      <ProjectsComponent
-        currentUser={props.me}
-        team={props.team}
-        savedSearches={props.team.saved_searches.edges.map(ss => ss.node)}
-        feeds={feeds.map(f => ({ ...f, title: (f.name || f.feed?.name), dbid: (f.feed_id || f.dbid) }))}
-      />
+      drawerType === 'default' ?
+        <ProjectsComponent
+          currentUser={props.me}
+          team={props.team}
+          savedSearches={props.team.saved_searches.edges.map(ss => ss.node)}
+          feeds={feeds.map(f => ({ ...f, title: (f.name || f.feed?.name), dbid: (f.feed_id || f.dbid) }))}
+        />
+        :
+        <FeedsComponent
```

Comment:
I think it's safer and more clear to explicitly check for `drawerType === 'feed'` here, so:

```jsx
if (drawerType === 'default') {
  return (<ProjectsComponent ... />);
} else if (drawerType === 'feed') {
  return (<FeedsComponent ... />);
}
return null;
```

Reply:
:white_check_mark: